app/assets/javascripts/administrate/components/belongs_to_form.js

```diff
@@ -0,0 +1,23 @@
+$(function() {
+  $(".field-unit--belongs-to select[data-remote-url]").each(function(index, field) {
+    $(field).selectize({
+      valueField: 'id',
```

Comment:
Why do we need `valueField`, `labelField` etc, and why those are set `id/name`?

Reply:
This is just telling the selectize plugin which fields from the json returned by the remote query to use for the value (valueField) and text (labelField) on the select.